e2e/heuristic_test.go

```diff
@@ -31,7 +31,7 @@ import (
 // balance enforcement heuristic session on L2 network.
 func TestBalanceEnforcement(t *testing.T) {
 
-	ts := e2e.CreateSysTestSuite(t)
+	ts := e2e.CreateSysTestSuite(t, "")
```

Comment:
Don't love the idea of extending this constructor with a single string value - is there not a way to encapsulate this?

Reply:
I agree, but I couldn't think of a better way to solve this. Open to any ideas here